referenceframe/model.go

```diff
@@ -296,3 +308,32 @@ func ModelFromPath(modelPath, name string) (Model, error) {
 	}
 	return model, err
 }
+
+// New2DMobileModelFrame builds the kinematic model associated with the kinematicWheeledBase
+// Note that this model is not intended to be registered in the frame system.
```

Comment:
With the robot's frame system service; it can (and will!) be registered with *a* frame system just fine.

Reply:
Ah yes this is a holdover from an old comment.  I will remove